api/inventorypb/agent_status.proto

```diff
@@ -11,12 +11,14 @@ enum AgentStatus {
   STARTING = 1;
   // Agent is running.
   RUNNING = 2;
-  // Agent encountered error and will be restarted automatically soon.
+  // Agent will be restarted automatically soon.
```

Comment:
it seems for me like it's still a true statement

Reply:
My primary aim here is to differentiate this error from an initialization error. How about "Agent encountered error when running and will be restarted automatically soon."